authorities.md

```diff
@@ -0,0 +1,95 @@
+# Authorities Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+## Main Endpoint
+**/api/integration/authorities**   
+
+Provide access to the configured authorities. It returns the list of existent authorities.
+
+Example: to be provided
+
+## Single Authority
+**/api/integration/authorities/<:authority-name>**
+
+Provide detailed information about a specific authority. The JSON response document is as follow
+```json
+{
+  "name": "SolrAuthorAuthority",
```

Comment:
Agreed with @tomdesair that this shouldn't be named "SolrAuthorAuthority". The frontend shouldn't care where this authority is stored, and shouldn't even know it has anything to do with Solr.

Reply:
The authority name comes from the configuration (https://github.com/DSpace/DSpace/blob/master/dspace/config/dspace.cfg#L1440), it doesn't imply that the frontend care about how this authority is implemented or the information stored. One authority plugin is intend to be used for multiple metadata, i.e. one can link the ORCID authority to dc.contributor.author, dc.contributor.editor, etc.
Of course it is possible to create some generic implementation of the authority, as in the case of the SOLRAuthority, where the available authority values change drastically from one metadata to another... personally I don't like this approach when used too widely, having dedicated authority for different concepts like persons, publishers, etc. allow the implementation/support at different levels of referential integrity. Anyway, the proposed contract allows us to use generic authorities (such as the SOLRAuthority) as the authority will receive the metadata field as a parameter for the /entries endpoint, see https://github.com/DSpace/Rest7Contract/pull/11/files#diff-6f10a81a770a59271e7d4ffdcfabf516R34 single purpose authorities like the ORCIDAuthority is most likely to ignore the metadata parameter as each researcher is a good candidate for both the author than the editor role

Reply:
@abollini : thanks for clarifying. Since this is coming out of the configuration, I think it's fine for now. I didn't realize where the `SolrAuthorAuthority` name was coming from (and wrongly assumed it was  coming directly from the class name).